practiceCourseGenerator/practiceCourseGenerator.py

```diff
@@ -163,10 +175,14 @@ def setupLogger(logdate):
 
 	return logger
 
-logdate = time.strftime("%Y%m%d%H%M")
-logger = setupLogger(logdate)
+print 'Number of Arguments: ', len(sys.argv), 'arguments.'
+print 'Arguments List:', str(sys.argv)
```

Comment:
Are these two print statements just for debugging purposes?  If so, remove them.